### ISSUE TITLE
When SKU are sync, WC throws the error "Invalid or duplicated SKU."

### ISSUE THREAD

**Issue by decarvalhoaa** [hyyan/woo-poly-integration#114]

**Comment by hyyan** [comment]
It seems this behavior has been changed in WC7,  this might be a problem we need to invistigate Carefully

**Comment by decarvalhoaa** [comment]
Fixed via [7977fdd](https://github.com/hyyan/woo-poly-integration/commit/7977fdd790afe62b33d63ab4e3673ce4d8aa1e51).

**Comment by orestis-s** [comment]
While the commit fixes the issue, this doesn't seem to be merged on the current release and the error keeps popping up.

Dirty fix till this is merged back to the release

```
add_filter('wc_product_has_unique_sku', 'suppressInvalidDuplicatedSKUErrorMsg', 100);
function suppressInvalidDuplicatedSKUErrorMsg($sku_found) {
    $metas = Hyyan\WPI\Product\Meta::getProductMetaToCopy();

    if (in_array('_sku', $metas))
        return false;
    
    return $sku_found;
}
```

**Comment by Jon007** [comment]
@orestis-s yes you are right, this was fixed in the unreleased woocommerce 2.7 branch and included in the woocommerce 3.0 release candidates under way at the moment at:
https://github.com/Jon007/woo-poly-integration/releases
